footer.html

```diff
@@ -39,7 +39,7 @@ <h3 class="footer-heading">Follow Us</h3>
                 </div>
                 <div class="col-6 col-sm-6 col-md-3">
                     <h3 class="footer-heading">Resources</h3>
-                    <ul class="list-unstyled text-small">
```

Comment:
It appears this change has added bullet points to the footer

Reply:
<img width="152" alt="image" src="https://user-images.githubusercontent.com/46505951/214901569-900e1066-5d55-4799-bb3d-ba62cb00c9a7.png">

Reply:
Fixed - that change has been reverted in favor of css changes